### DISCUSSION TITLE
About ID verification

### DISCUSSION THREAD

**Discussion by Sandesh9892** [community/community#108320] [GitHub Education]
### Select Topic Area

Question

### Body

I have been trying to submit my school ID for the student package but my school does not have a Validity date what other document should I put to get the package?

**Comment by Ariffazeel99** [comment]
you can try with Official Email Address provided by school

**Reply by John-Polo** [reply]
@Ariffazeel99 I have an official school email address and use it as default in the profile for my account. It doesn't seem to help me. Is there something else to do besides having that in the profile that makes it acceptable for the verification?

**Reply by Sandesh9892** [reply]
I have an official email address given by my university.
